src/components/Icon.vue

```diff
@@ -3,6 +3,7 @@
     class="iconfont"
     :class="`icon${name}`"
     :style="size ? `font-size: ${size}px; line-height: ${size}px;` : ''"
+    @click="$emit('click')"
```

Comment:
You shouldn't change this component, if you want to implement a click you can a link around the icon like this 
```
<a href="#" @click=""><Icon .../></a>
```